code/datums/components/chem_booster.dm

```diff
@@ -0,0 +1,210 @@
+/**
+	Chem booster component
+
+	This component stores "green blood" and uses it to increase effect_str of chems in the user's body.
+
```

Comment:
Autodoc can work on types too so make this doc proper if you don't mind

Reply:
changed